src/util/misc.ts

```diff
@@ -205,6 +228,12 @@ export function genHeaderScript(config: Config, js: string, articleId: string, s
   const path = isNodeModule(js) ? normalizeModule(js) : js
   return `<script ${attributes} src="${upStr}${resourceNamespace}/${jsPath(path, subDirectory)}"></script>`
 }
+export function genPCSOverrideScript(js: string) {
+  return `<script src='../-/${js}.js'></script>`
+}
+export function genPCSCOverrideCSSLink(css: string) {
+  return `<link rel="stylesheet" href="../-/${css}.css" />`
+}
```

Comment:
This is very specific to the wikimediamobiler render, and should be only there.

Reply:
Done, I replaced this to the `src/renderers/abstractMobile.render.ts`